### PR TITLE
Add parity signer & account tutorials

### PR DESCRIPTION
**Pull request by EugenWay**
Resolves # .

-
-
-

@reviewer-or-team

### CODE REVIEW

File: docs/general/via-parity-signer.mdx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+Go offline, generate and store your private keys. Backup your seed phrase on paper or try other reliable methods.
+
+## Add Vara Network
+
+To sign messages/transactions, and transfer funds to and from VARA account We need to add Vara Network to app.
+
+1. Open your Signer and select tab `scan`
+
+2. Go to [Metadata Portal](https://metadata.vara-network.io/#/vara)
```
Comment (breathx) [inline docs/general/via-parity-signer.mdx:49]: and open chain specs tab*

File: docs/general/via-parity-signer.mdx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+## How to use
+
+The signer allows you to sign transactions using an air-gapped way. You scan a QR code with details about your transaction, then generates a code already signed on your offline device and deliver it as confirmation. Many applications support this feature.
+
+### Polkadot.{js}
+
+To import your hard wallet account into Polkadot.js:
+
+1. You should go to (account section)[https://polkadot.js.org/apps/#/accounts] and click `from QR` button.
```
Comment (breathx) [inline docs/general/via-parity-signer.mdx:81]:
```suggestion
1. You should go to [account section](https://polkadot.js.org/apps/#/accounts) and click `from QR` button.
```
I'd also provide the screenshot of that page

File: docs/general/via-parity-signer.mdx [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+Done! Now, when making transactions from this account, we will sign them with your Signer using QR codes.
+
+### Polkadot extension
+
+1. In your Polkadot extension allow QR camera access
+
+![add metadata](./img/p-ext-1.png)
+
+2. Click + to import new accoutn and choose `Attach external QR-signer account` option:
```
Comment (breathx) [inline docs/general/via-parity-signer.mdx:99]:
```suggestion
2. Click + to import new account and choose `Attach external QR-signer account` option:
```

File: docs/general/via-parity-signer.mdx [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+  </a>
+</div>
+
+Download directly from the github:
+
+https://github.com/paritytech/parity-signer/releases
+
+## Create your keys
+
+:::info
```
Comment (breathx) [inline docs/general/via-parity-signer.mdx:37]: Maybe some screenshots here with explanation how single seed phrase can be used with any amount of public addresses via salt?

File: docs/general/create-account.mdx [hunk 5]
```diff
@@ -0,0 +1,142 @@
+---
+title: Create account
+sidebar_position: 1
+sidebar_label: Create account
+---
+
+The Vara Network is substrate-based, which means that any keypair compatible with the standards of the `subkey` module will also work on Vara.
```
Comment (breathx) [inline docs/general/create-account.mdx:7]: in fact address spaces depends only on crypto curves and for our case its common for substrate sr25519

File: docs/general/create-account.mdx [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+
+![img alt](./img/create-account-3.png)
+
+:::note
+The password that you choose here will be used to encrypt this account's information. You will need to re-enter it when attempting to process any kind of outgoing transaction, or when using it to cryptographically sign a message. This password is stored locally on the browser extension.
+:::
+
+You've now successfully created a new account using the polkadot.js browser extension.
+
+### Via mobile app
```
Comment (breathx) [inline docs/general/create-account.mdx:102]: This one and further one have other level of headers, seem should be like this
 ```suggestion
## Via mobile app
```

File: docs/general/create-account.mdx [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+:::
+
+:::info
+
+### Storing your accounts JSON file
+
+Your account’s JSON file is encrypted with a password. This means that while you can import into any wallet, you will need to use the password to do so successfully. As there is this additional layer of security - a password - you don’t have to be as cautious about securing it. However, it’s worth using highly secure passwords that are not used for anything else. It would be best if you made a note of this password offline.
+:::
+
+## Via Talisman
```
Comment (breathx) [inline docs/general/create-account.mdx:30]: I'd place polkadot extension first as most popular variant

**Review by breathx — APPROVED** [review]

File: docs/general/create-account.mdx [hunk 8]
```diff
@@ -0,0 +1,142 @@
...
+
+3. Click the large plus button to create a new account. Alternatively, you can navigate to the smaller plus icon in the top right and select "Create New Account".
+
+![img alt](./img/create-account-1.png)
+
+4. Save the `seed phrase` in a safe place using the security advices at the beginning of this article.
+
+![img alt](./img/create-account-2.png)
+
+5.  Specify the name of your account and a strong password for making transactions. Click "Add the account with the generated seed".
```
Comment (shamilsan) [inline docs/general/create-account.mdx:92]:
```suggestion
5. Specify the name of your account and a strong password for making transactions. Click "Add the account with the generated seed".
```

File: docs/general/via-parity-signer.mdx [hunk 9]
```diff
@@ -0,0 +1,109 @@
...
+sidebar_position: 2
+sidebar_label: Via Parity Signer
+---
+
+import googleURL from './img/google-play-badge.png';
+import appleURL from './img/app-store-badge.png';
+
+# Vara cold storage via Parity Signer
+
+Parity Signer allows you to turn any mobile device based on Android or iOS into a real cold walllet. Your private keys are kept secure offline at all times and transactions are signed in an air-gapped way via QR-codes. A huge advantage is the completely open and transparent source code of [the product](https://github.com/paritytech/parity-signer)
```
Comment (shamilsan) [inline docs/general/via-parity-signer.mdx:12]:
```suggestion
Parity Signer allows you to turn any mobile device based on Android or iOS into a real cold wallet. Your private keys are kept secure offline at all times and transactions are signed in an air-gapped way via QR-codes. A huge advantage is the completely open and transparent source code of [the product](https://github.com/paritytech/parity-signer)
```

File: docs/general/via-parity-signer.mdx [hunk 10]
```diff
@@ -0,0 +1,109 @@
...
+
+:::info
+Once Parity Signer is installed, your device should never go online. You must turn off or even physically remove the smartphone's Wifi, Mobile Network, and Bluetooth to ensure that the mobile phone containing these accounts will not be exposed to any online threat. Switching to airplane mode suffices in many cases.
+:::
+
+Go offline, generate and store your private keys. Backup your seed phrase on paper or try other reliable methods.
+
+## Add Vara Network
+
+To sign messages/transactions, and transfer funds to and from VARA account We need to add Vara Network to app.
```
Comment (shamilsan) [inline docs/general/via-parity-signer.mdx:45]:
```suggestion
To sign messages/transactions, and transfer funds to and from VARA account, we need to add Vara Network to app.
```

File: docs/general/via-parity-signer.mdx [hunk 11]
```diff
@@ -0,0 +1,109 @@
...
+To get the latest blockchain metadata:
+
+1. Open your Signer and select tab `scan`
+
+2. Again go to [Metadata Portal](https://metadata.vara-network.io/#/vara)
+
+3. Scan with your device video QR with matadata frames
+
+:::info
+Scanning QR with metadata could take some time. Remember to update the Signer to the latest version of metadata, you will need to do the scan again. We generate new QR's with a each runtime upgrade of the network.
```
Comment (shamilsan) [inline docs/general/via-parity-signer.mdx:68]:
```suggestion
Scanning QR with metadata could take some time. Remember to update the Signer to the latest version of metadata, you will need to do the scan again. We generate new QR's with each runtime upgrade of the network.
```

File: docs/general/via-parity-signer.mdx (hunk 3 above)
Comment (shamilsan) [inline docs/general/via-parity-signer.mdx:99]:
```suggestion
2. Click <keyboard>+</keyboard> to import new account and choose `Attach external QR-signer account` option:
```

File: docs/general/via-parity-signer.mdx [hunk 12]
```diff
@@ -0,0 +1,109 @@
...
+
+There are several ways to get the Parity Signer
+
+Download from the application store for your device:
+
+<div align="left">
+  <a href="https://play.google.com/store/apps/details?id=io.parity.signer">
+    <img src={googleURL} alt="Play Market" />
+  </a>{' '}
+  <a href="https://itunes.apple.com/us/app/parity-signer/id1218174838">
```
[the remark this answers is not in the document]
Comment (EugenWay) [reply docs/general/via-parity-signer.mdx:26]: strange white line XD

**Review by shamilsan — APPROVED** [review]
Excellent 🌟

**Review by AndrePanin — APPROVED** [review]

**Pull request merged**